pkg/storage/secret/metadata/keeper_model.go

```diff
@@ -103,7 +100,7 @@ func toKeeperCreateRow(kp *secretv0alpha1.Keeper, actorUID string) (*keeperDB, e
 		return nil, fmt.Errorf("failed to map to row: %w", err)
 	}
 
-	now := time.Now().UTC().UnixMilli()
+	now := time.Now().UTC().Unix()
```

Comment:
This fixes the weird timestamps we were seeing, sneaked in here to pass the integration tests properly...